pkg/features/kube_features.go

```diff
@@ -1017,7 +1018,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	RotateKubeletServerCertificate: {Default: true, PreRelease: featuregate.Beta},
 
-	SeccompDefault: {Default: true, PreRelease: featuregate.Beta},
+	SeccompDefault: {Default: true, PreRelease: featuregate.GA},
```

Comment:
should be locked to default as per https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-stages

Reply:
```suggestion
	SeccompDefault: {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
```

Reply:
pease also remove all the usage of this feature gate in code